apps/website/src/pages/ambassadors/index.tsx

```diff
@@ -155,24 +163,30 @@ const AmbassadorGroup: React.FC<{
     <div ref={scroll}>
       <Heading
         level={2}
-        className="mb-4"
+        className="mb-8"
         id={`${type}:${camelToKebab(group)}`}
         link
       >
         {name}
       </Heading>
-      <AmbassadorItems
-        ambassadors={ambassadors}
-        className="md:ml-8"
-        level={3}
-      />
+      <div className="grid gap-8 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-6">
+        {ambassadors.map((key) => (
+          <AmbassadorItem key={key} ambassador={key} level={3} />
+        ))}
+      </div>
     </div>
   );
 };
 
-const AmbassadorsPage: NextPage = () => {
+export const getStaticProps: GetStaticProps = async () => {
```

Comment:
Is there a reason to start using static props here? Seems like we're just going to be generating extra data in the initial payload as this will be in the JS data as well as the JSON data now?